canonical-websites/design.ubuntu.com
# Remove dot patterns page.

## Done

Removed the dot pattern page

## QA

 - `./run`
 - see that `/brand/dot-patterns` no longer exists and that there are no references to it (e.g. on `/brand`)
